streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java

```diff
@@ -598,9 +598,7 @@ public void updateHotParams(Application appParam) {
     ExecutionMode executionModeEnum = appParam.getExecutionModeEnum();
     Map<String, String> hotParams = new HashMap<>(0);
     if (ExecutionMode.YARN_APPLICATION.equals(executionModeEnum)) {
-      if (StringUtils.isNotEmpty(appParam.getYarnQueue())) {
-        hotParams.put(ConfigConst.KEY_YARN_APP_QUEUE(), appParam.getYarnQueue());
-      }
+      YarnQueueLabelExpression.addQueueLabelExprInto(appParam.getYarnQueue(), hotParams);
```

Comment:
`ExecutionMode.YARN_APPLICATION.equals(executionModeEnum)` need to be changed to:
`ExecutionMode.isYarnMode(executionModeEnum)`

This is a historical bug